services/minespace-web/src/constants/routes.js

```diff
@@ -25,6 +26,25 @@ export const USERS = {
   component: UsersPage,
 };
 
+export const ADD_PROJECT_SUMMARY = {
+  route: "/mines/:mineGuid/project-summary/new",
+  dynamicRoute: (mineGuid) =>
+    `/mines/${mineGuid}/project-summary/new`,
+  component: ProjectSummaryPage,
+}
+
+export const EDIT_PROJECT_SUMMARY = {
+  route: "/mines/:mineGuid/project-summary/:projectSummaryGuid",
+  dynamicRoute: (mineGuid, projectSummaryGuid) =>
+    `/mines/${mineGuid}/project-summary/${projectSummaryGuid}`,
+  component: ProjectSummaryPage,
+}
+
+export const MINE_PROJECT_SUMMARIES = {
+  dynamicRoute: (mineGuid) =>
+    `/mines/${mineGuid}/projectSummaries`
+}
+
```

Comment:
'MINE_PROJECT_SUMMARIES' is this the tab? if so, the 'MINE_DASHBOARD' route should work here, and the ":activeTab" === "projectSummaries"